frontend/src/css/App.css

```diff
@@ -7,3 +7,84 @@
   right: 0;
   padding: 4px 4px 0 0;
 }
+
```

Comment:
We should use comments to section this CSS document since we're keeping it all together.

`// Tabs` before the section would be sufficient.

Reply:
Added in commit d7c042f